test/matrix/fft_kernels.cpp

```diff
@@ -160,6 +163,9 @@ TYPED_TEST(Fft, Apply1DInverseIsEqualToReference)
 
 TYPED_TEST(Fft, ApplyStrided1DInverseIsEqualToReference)
 {
+#if defined(GKO_COMPILING_HIP) && GINKGO_HIP_PLATFORM_HCC
+    GTEST_SKIP() << "rocFFT has a bug related to strided 1D FFT";
+#endif
```

Comment:
is something related to this https://github.com/ROCmSoftwarePlatform/rocFFT/issues/353 ?
but it should pass all in the current CI setup

Reply:
No, this is a different bug (causing incorrect results, not an error) I didn't bother creating a reproducer for yet.